### ISSUE TITLE
"loadbalancer member set" does nothing when weight is 0

### ISSUE THREAD

**Issue by njgte** [opentelekomcloud/python-otcextensions#242]
I want to use the OpenStack Client to update the weight of a member of an elastic load balancer which is part of an open telekom cloud instance. I can successfully set the weight to 1, or other positive numbers using the cli. However it doesn't work for the weight 0. 

**Versions and how I installed**

I am working in an virtual environment using python version 3.8.10. Inside of this environment I installed openstack, as well as the otc-extensions using: 

```shell
pip install python-openstackclient
pip install python-openstacksdk otcextensions
```

The installed versions are openstack 5.6.0, with otcextensions version 0.24.0. 

**Problem Description**

I am using a clouds.yaml file with ak/sk for authentification. The exact command that I had the problem with is: 

```shell
openstack --os-cloud otc loadbalancer member set --weight 0 "$POOL_ID" "$MEMBER_ID"
```

**Expected outcome**
I expected the specified member to have the weight 0 afterwards. I also expected the output of the command to be similar to the output when I use a weight > 0. 

**Actual outcome**
After running the command the weight of the member remains unchanged (I checked this in the web interface)
The output of the command is the following: 

```
+----------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Field          | Value                                                                                                                                                                                     |
+----------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| admin_state_up |                                                                                                                                                                                           |
| id             | -here was a valid ID-                                                                                                                                                       |
| location       | Munch({'cloud': 'otc', 'region_name': 'eu-de', 'zone': None, 'project': Munch({'id': 'here was a valid id, 'name': 'eu-de', 'domain_id': 'default', 'domain_name': None})}) |
+----------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+ 
```

This is the same output, as when I run the command completely without the weight option, so: `openstack --os-cloud otc loadbalancer member set "$POOL_ID" "$MEMBER_ID"`

Please let me know if you need any more information

[Here is the log output, when --debug option is specified. ](https://github.com/opentelekomcloud/python-otcextensions/files/7453309/debug.log)

**Comment by anton-sidelnikov** [comment]
@noahjgente Hello, can you attach --debug log?

**Comment by njgte** [comment]
Added the log

**Comment by anton-sidelnikov** [comment]
Released 0.24.2 with fix
